<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalserverWindow</class>
 <widget class="QMainWindow" name="LocalserverWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Powerd By rajIT Ltd.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="syncButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sync</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>321</width>
      <height>341</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>331</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="statusAction"/>
    <addaction name="actionAdd_New"/>
    <addaction name="listAction"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionConfigaration"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="separator"/>
    <addaction name="versionAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionConfigaration">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icon/feather/settings.svg</normaloff>:/icon/feather/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Configaration</string>
   </property>
  </action>
  <action name="actionAdd_New">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icon/feather/plus.svg</normaloff>:/icon/feather/plus.svg</iconset>
   </property>
   <property name="text">
    <string>Add New Device</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>fugue-icons-3.5.6/icons/document-sub.png</normaloff>fugue-icons-3.5.6/icons/document-sub.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_list">
   <property name="icon">
    <iconset>
     <normaloff>fugue-icons-3.5.6/icons/document-share.png</normaloff>fugue-icons-3.5.6/icons/document-share.png</iconset>
   </property>
   <property name="text">
    <string>Device List</string>
   </property>
  </action>
  <action name="actionConfigaration_2">
   <property name="icon">
    <iconset>
     <normaloff>fugue-icons-3.5.6/icons/hammer-screwdriver.png</normaloff>fugue-icons-3.5.6/icons/hammer-screwdriver.png</iconset>
   </property>
   <property name="text">
    <string>Configaration</string>
   </property>
  </action>
  <action name="versionAction">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icon/feather/help-circle.svg</normaloff>:/icon/feather/help-circle.svg</iconset>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="statusAction">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icon/feather/info.svg</normaloff>:/icon/feather/info.svg</iconset>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="listAction">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icon/feather/list.svg</normaloff>:/icon/feather/list.svg</iconset>
   </property>
   <property name="text">
    <string>Device List</string>
   </property>
   <property name="toolTip">
    <string>list</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
